src/synchronizeContacts.js

```diff
@@ -71,6 +73,7 @@ const synchronizeContacts = async (
         transpiledContact,
         mergeWithCozyOverride
       )
+      unset(merged, 'trashed')
```

Comment:
Nothing to do with this PR: I think that the code of the contacts app rely on the fact that we never have `trashed: false`. We should fix this (or at least create a card to keep a trace of this "issue")

Just checked, in drive I did: 

```
      trashed: {
        $or: [{ $eq: false }, { $exists: false }]
      },
```